sds_data_manager/constructs/instrument_lambdas.py

```diff
@@ -18,14 +19,16 @@ def __init__(
         self,
         scope: Construct,
         construct_id: str,
-        processing_step_name: str,
         data_bucket: s3.Bucket,
         code_path: str or Path,
-        instrument_target: str,
-        instrument_sources: str,
+        instrument: str,
+        instrument_dependents: dict,
```

Comment:
rename to `instrument_downstream` (or `instrument_upstream` if it's some kind of mapping of instruments to upstream dependencies?)